addon/gql/queries/get-fileanswer-info.graphql

```diff
@@ -0,0 +1,14 @@
+query getFileAnswerInfo($id: ID!) {
+  node(id: $id) {
+    id
+
```

Comment:
nitpick: whitespace

Reply:
Whitespace? The empty line after `id`?

Reply:
yep - or is this something we do anywhere else already?

Reply:
I'll have to check. It is something I do elsewhere :smile:

Reply:
We don't. Change is coming with the next commit.